src/isomorphic/classic/element/ReactElement.js

```diff
@@ -15,6 +15,11 @@ var ReactCurrentOwner = require('ReactCurrentOwner');
 
 var assign = require('Object.assign');
 
+// The Symbol used to tag the ReactElement type. If there is no native Symbol
+// nor polyfill, then a plain number is used for performance.
+var TYPE_SYMBOL = (typeof Symbol === 'function' && Symbol.for &&
+                  Symbol.for('react.element')) || 0xeac7;
```

Comment:
See #3473 for more context.

It is too easy to expect a string and not realize it might be an object:

```js
loadString(function(stringData) {
  React.render(<div>Content: {stringData}</div>, container);
});
```